src/main/kotlin/org/rust/lang/core/psi/RsDefaultValueBuilder.kt

```diff
@@ -90,10 +90,10 @@ class RsDefaultValueBuilder(
         structLiteral: RsStructLiteralBody,
         declaredFields: List<RsFieldDecl>,
         fieldsToAdd: List<RsFieldDecl>
-    ): RsStructLiteralField? {
+    ): List<RsStructLiteralField> {
```

Comment:
Maybe collect `List< SmartPsiElementPointer< RsStructLiteralField>>` right here? I'm not sure that these elements didn't invalidate before the creation of a smart pointer.